### ISSUE TITLE
Failed to build latest with gcc 11.2

### ISSUE THREAD

**Issue by bezirg** [L-as/procex#1]
I am on archlinux x64, gcc 11.2 and I get this error output:

```bash
$ cabal build procex:procex
Build profile: -w ghc-8.10.7 -O1
In order, the following will be built (use -v for more details):
 - procex-0.3.1 (lib) (first run)
Preprocessing library for procex-0.3.1..
Building library for procex-0.3.1..

cbits/glue.c:13:12: error:
     error: conflicting types for ‘close_range’; have ‘int(unsigned int)’
       13 | static int close_range(unsigned int first) {
          |            ^~~~~~~~~~~
   |
13 | static int close_range(unsigned int first) {
   |            ^

In file included from cbits/glue.c:7:0: error: 

/usr/include/unistd.h:1208:12: error:
     note: previous declaration of ‘close_range’ with type ‘int(unsigned int,  unsigned int,  int)’
     1208 | extern int close_range (unsigned int __fd, unsigned int __max_fd,
          |            ^~~~~~~~~~~
     |
1208 | extern int close_range (unsigned int __fd, unsigned int __max_fd,
     |            ^
`gcc' failed in phase `C Compiler'. (Exit code: 1)
```

The same error happens both for ghc 8.10.7 and ghc 9.0.1 (both installed via ghcup)

**Comment by L-as** [comment]
Thank you for reporting this, I fixed this in acd13d69ca10165264f88a80c03d039ec718ff5e.

**Comment by L-as** [comment]
This seems to be because close_range is now in unistd. I wonder whether there's a way to ignore that definition. Mine is static after all!
